src/app/modules/MainScreen/MainScreen.tsx

```diff
@@ -1,9 +1,13 @@
 import React, { PureComponent } from 'react';
-import { MainWrapper, MainTitle } from './MainScreen.styles';
+import { MainWrapper, MainTitle } from './styles';
 import { Button } from 'react-native';
 import Modules from '../index';
 
-class MainScreen extends PureComponent {
+interface IProps {
+  navigation: any;
```

Comment:
Probably navigation props types can be imported. Please remove any.
https://reactnavigation.org/docs/en/typescript.html